align_system/utils/alignment_utils.py

```diff
@@ -79,8 +85,23 @@ def __call__(self, kdma_values, target_kdmas, misaligned=False):
             distances.append(distance)
 
         selected_choice, probs = self._select_min_dist_choice(choices, distances, misaligned)
-
         return selected_choice, probs
+    
+    def get_best_sample_index(self, kdma_values, target_kdmas, selected_choice, misaligned=False, kde_norm=None):
```

Comment:
is the `kde_norm` argument being used here?

Reply:
It is not, fixed here https://github.com/ITM-Kitware/align-system/pull/98/commits/f063db1e7029ec986df50496491e0b6902854656